client/components/homepage/CreateRoute.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
+import io from 'socket.io-client';
```

Comment:
Only thing I'm wondering is whether or not we should just require socket.io on the homepage, and then pass the socket function down to create room, and then to room and it's children (since adding functions to the store is frowned upon).